api-report/fluid-framework.api.md

```diff
@@ -5,15 +5,259 @@
+import { IntervalType } from '@fluidframework/sequence';
+import { IRootDataObject } from '@fluidframework/fluid-static';
+import { ISequenceDeltaRange } from '@fluidframework/sequence';
+import { ISerializableInterval } from '@fluidframework/sequence';
+import { ISerializableValue } from '@fluidframework/map';
+import { ISerializedInterval } from '@fluidframework/sequence';
+import { ISerializedIntervalCollectionV2 } from '@fluidframework/sequence';
+import { ISerializedValue } from '@fluidframework/map';
+import { IServiceAudience } from '@fluidframework/fluid-static';
+import { IServiceAudienceEvents } from '@fluidframework/fluid-static';
+import { ISharedDirectory } from '@fluidframework/map';
+import { ISharedDirectoryEvents } from '@fluidframework/map';
+import { ISharedIntervalCollection } from '@fluidframework/sequence';
+import { ISharedMap } from '@fluidframework/map';
+import { ISharedMapEvents } from '@fluidframework/map';
+import { ISharedSegmentSequenceEvents } from '@fluidframework/sequence';
+import { ISharedString } from '@fluidframework/sequence';
+import { IValueChanged } from '@fluidframework/map';
+import { IValueOpEmitter } from '@fluidframework/sequence';
+import { LoadableObjectClass } from '@fluidframework/fluid-static';
+import { LoadableObjectClassRecord } from '@fluidframework/fluid-static';
+import { LoadableObjectCtor } from '@fluidframework/fluid-static';
+import { LoadableObjectRecord } from '@fluidframework/fluid-static';
+import { LocalValueMaker } from '@fluidframework/map';
+import { MapFactory } from '@fluidframework/map';
+import { MemberChangedListener } from '@fluidframework/fluid-static';
+import { RootDataObject } from '@fluidframework/fluid-static';
+import { RootDataObjectProps } from '@fluidframework/fluid-static';
+import { SequenceDeltaEvent } from '@fluidframework/sequence';
+import { SequenceEvent } from '@fluidframework/sequence';
+import { SequenceInterval } from '@fluidframework/sequence';
+import { SequenceMaintenanceEvent } from '@fluidframework/sequence';
+import { SerializedIntervalDelta } from '@fluidframework/sequence';
+import { ServiceAudience } from '@fluidframework/fluid-static';
+import { SharedDirectory } from '@fluidframework/map';
+import { SharedIntervalCollection } from '@fluidframework/sequence';
+import { SharedIntervalCollectionFactory } from '@fluidframework/sequence';
+import { SharedMap } from '@fluidframework/map';
+import { SharedObjectClass } from '@fluidframework/fluid-static';
+import { SharedSegmentSequence } from '@fluidframework/sequence';
+import { SharedSequence } from '@fluidframework/sequence';
+import { SharedString } from '@fluidframework/sequence';
+import { SharedStringFactory } from '@fluidframework/sequence';
+import { SharedStringSegment } from '@fluidframework/sequence';
+import { SubSequence } from '@fluidframework/sequence';
 
 export { AttachState }
 
+export { CompressedSerializedInterval }
```

Comment:
do we really need an export per type? can't they all be bundled into a single export statement?

Reply:
ahh. this is the mark down. blah.

Reply:
however, this file is insane. fyi @tylerbutler @Josmithr who i think know about docs

Reply:
Yeah, it's just that this what gets generated by the api-extractor after I run "npm run build" after my script makes the changes to the ts files. I never directly touch the .md files!

Reply:
If I keep the export * in this file, the CI fails to build it.